debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/BinlogReaderBufferIT.java

```diff
@@ -349,4 +356,15 @@ public void shouldProcessRolledBackSavepoint() throws SQLException, InterruptedE
             Testing.print("*** Done with savepoint TX");
         }
     }
+
+    private static boolean isMySql5(String databaseName) {
```

Comment:
Would it it be better to move this method into `src/test/java/io/debezium/connector/mysql/MySQLConnection.java` ?

Reply:
Probably, I'll move it.